.github/workflows/verify.yml

```diff
@@ -33,7 +33,7 @@ jobs:
     name: Docker Build
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
v2 is also present in other Github action files within this repo, which I imagine we'll want to update too?

Reply:
Yep let me take a look for those and update them as well.